CONTRIBUTING.md

```diff
@@ -102,6 +103,15 @@ Closes #318
 Update **`Start working on Issues`** section with clearer instructions on getting ready to work on an issue.
 ```
 
+### Disabling GitHub Action
```

Comment:
Please use `GitHub Actions` consistently across the documents.
```suggestion
### Disabling GitHub Actions
```

Reply:
I just push your suggestion. thanks